### PR TITLE
Feature/linkonrow

### PR DESCRIPTION
**Pull request by wilsoncooked**
Chane link to the details page for the whole row using an on click event

### CODE REVIEW

File: src/components/global/ActionsTable.js [hunk 1]
```diff
@@ -5,13 +5,18 @@ import {Link} from 'react-router-dom';
 import {withRouter} from 'react-router-dom';
 
 function ActionsTable(props) {
+  const handleClick = (pathName, id) => {
+    props.history.push(`/${pathName}/${id}`);
```
Comment (laura-mso) [inline src/components/global/ActionsTable.js]:
```suggestion
    if (pathName !== 'users') {props.history.push(`/${pathName}/${id}`);}
```

File: src/components/project/ClientInfo.js [hunk 2]
```diff
@@ -44,17 +44,17 @@ export default class ClientInfo extends React.Component {
...
+                  <img
+                    className='card-img-top client-logo'
+                    src={this.state.client.logo}
+                    alt='client logo'
+                  />
+                ) : (
+                  <i className='material-icons client-placeholder'>
+                    location_city
+                  </i>
+                )}
```
Comment (laura-mso) [inline src/components/project/ClientInfo.js:57]: thanks for fixing this!

File: src/components/global/ActionsTable.js [hunk 3]
```diff
@@ -46,11 +51,19 @@ function ActionsTable(props) {
               {tableData !== undefined && tableData.length > 0 ? (
                 tableData.map((dataRow, rowIndex) => {
                   return (
-                    <tr key={rowIndex}>
+                    <tr key={rowIndex} className='tr-mouse'>
```
Comment (laura-mso) [inline src/components/global/ActionsTable.js]:
```suggestion
                    <tr key={rowIndex} className={(pathname === 'users' ? ' ' : 'tr-mouse')}>
```

File: src/components/task/TasksTable.js [hunk 4]
```diff
@@ -100,54 +118,74 @@ export default class TasksTable extends React.Component {
...
-        }
-      />,
-    ]);
+    var orderedTasks = this.state.tasks,
+      tasksByProjects = orderedTasks.reduce((accumulator, taskObject) => {
+        accumulator[taskObject.projectId] =
+          accumulator[taskObject.projectId] || [];
+        accumulator[taskObject.projectId].push(taskObject);
+        return accumulator;
+      }, Object.create(null));
```
Comment (laura-mso) [inline src/components/task/TasksTable.js]: im a bit confused, why is this stuff in this pull request?

**Review by laura-mso — CHANGES_REQUESTED** [review]

File: src/components/task/TasksTable.js (hunk 4 above)
Comment (wilsoncooked) [reply src/components/task/TasksTable.js]: Oh yeah! Let me merge develop and it will fix it because I suck at git merging

**Review by laura-mso — CHANGES_REQUESTED** [review]
Its not working on the tasks page :((

**Comment by wilsoncooked** [comment]
Fixed tasks table

**Review by laura-mso — COMMENTED** [review]
Well done!!!!

**Review by laura-mso — DISMISSED** [review]

**Review by aliashkevich — APPROVED** [review]

**Pull request merged**